libnetwork/drivers/overlay/overlay.go

```diff
@@ -53,8 +52,7 @@ type driver struct {
 	joinOnce         sync.Once
 	localJoinOnce    sync.Once
 	keys             []*key
-	peerOpCh         chan *peerOperation
-	peerOpCancel     context.CancelFunc
+	peerOpMu         sync.Mutex
 	sync.Mutex
```

Comment:
A mutex on a struct which is extending mutex :yum:

Would it blow up the diff too much to move this second mutex into a field?

Reply:
Yes, and more importantly, conflict heavily with #44965